src/soroban_rpc.ts

```diff
@@ -64,6 +64,28 @@ export namespace SorobanRpc {
     error?: jsonrpc.Error;
   }
 
+  export interface EventFilter {
+    type?: string;
+    contractIds?: string[];
+    topics?: string[][];
+  }
+
+  export interface GetEventsResponse {
+    events?: EventResponse[];
```

Comment:
pagination?

Reply:
Let's either implement it, or let's un-document it if that is easier for now. (There was a comment that says it is supported)

Reply:
Added in latest push :+1: